Cargo.toml

```diff
@@ -8,3 +8,7 @@ members = [
     "examples",
 ]
 resolver = "2"
+
+[patch."crates-io"]
+quinn = { git = "https://github.com/quinn-rs/quinn", branch = "no-default-features" }
```

Comment:
@djc i think the branch `no-default-features` is not available anymore.
Should i test with `main` or do you have a specific branch?

Reply:
Main is good -- the branch has been merged. Thanks for having a look!